lib/lotus/commands/db.rb

```diff
@@ -0,0 +1,59 @@
+require 'lotus/router'
+require 'lotus/model'
+
+module Lotus
+  module Commands
+    class DB
+      attr_reader :environment, :options, :name
+      DEFAULTS = { step:1 }
+
+      def initialize(name=nil,environment)
+        @name = name
+        @environment = environment
+        @options = merge_options(@environment)
+      end
+
+      def migrate
+        load_environment
+        migrator.migrate
+      end
+
+      def rollback
+        load_environment
+        migrator.rollback(step: step)
+      end
+
+      private 
+
+      def config
+        if name
+          app_constant = Lotus::Utils::Class.load_from_pattern!(Lotus::Utils::String.new(name).classify)
+          Lotus::Utils::Class.load_from_pattern!("#{app_constant}::Application").load!
+          Lotus::Utils::Class.load_from_pattern!("#{app_constant}::Model").configuration
+        else
+          Lotus::Model.configuration
+        end
+      end
+
+      def step
+        options.fetch(:step) 
```

Comment:
what do you think about setting default value for step here instead? that is `options.fetch(:step, DEFAULT_STEP)`